nilearn/plotting/tests/test_js_plotting_utils.py

```diff
@@ -310,6 +319,10 @@ def test_temp_file_removing():
             os.remove(html._temp_file)
         except Exception:
             pass
+        try:
+            os.remove(tmpfile)
+        except Exception:
+            pass
 
```

Comment:
How do we know when this test fails? All the exceptions are silently passed.

Reply:
it's not a test, it is just cleaning up after the test